torch_xla/experimental/xla_quantized_matmul.py

```diff
@@ -14,17 +14,59 @@ def _check_per_channel_quant_weight_dtype_shapes(input_dim, output_dim, w,
   assert w.dtype == torch.int8, f"Weight dtype is expected to be torch.int8, got {w.dtype}."
   assert w.dim(
   ) == 2, f"Weight tensor is expected to be 2D, got {w.dim()}D Tensor."
-  assert output_dim == w.shape[0] and input_dim == w.shape[
-      1], f"Weight shape is expected to be [output_dim, input_dim], output_dim: {output_dim}, input_dim: {input_dim}, but got {w.shape}."
-  assert w_scaler.dim() == 1 and w_scaler.shape[0] == w.shape[
-      0], f"weight scaler shape is expect to be [out_channel,], got {w_scaler.shape}, weight shape {w.shape}."
+  w_shape = list(w.shape)
+  assert output_dim == w_shape[0] and input_dim == w_shape[
+      1], f"Weight shape is expected to be [output_dim, input_dim], output_dim: {output_dim}, input_dim: {input_dim}, but got {w_shape}."
+  assert w_scaler.dim() == 1 and w_scaler.shape[0] == w_shape[
+      0], f"weight scaler shape is expect to be [out_channel,], got {w_scaler.shape}, weight shape {w_shape}."
+
+
+def pack_4bit(x, dtype=torch.int8):
+  # Not used now, we don't do packed weight because reinterpret_cast
```

Comment:
don't merge the code that's not used, consider just delete them from this pr.

Reply:
Ok, I'll remove